### PR TITLE
Remakes Stealth's AshWalker/Lavaland stuff

### PR DESCRIPTION
**Pull request by X-TheDark**
Remakes #349 

I'm pretty sure everyone is going to cry about how overpowered Marrow Weavers are going to be.

Some fixes:
- Removes all things related to chat, less clutter in chat window for observers/dead folks;
- Removes alot of useless code used in Marrow Weavers (it's basically a copy-paste of spider code);
- Doubles the move delay for Marrow Weavers (was 3), similar to how it is with Fat Spider things. No, seriously, a 200 health mob that does 18 damage and injects 2 units of venom each bite that moved faster than you :+1: ;
- Removes their ability to wrap stuff, it's really more annoying than anything, considering these things don't lay eggs;

:cl: Stealthkibbler
rscadd: Ashwalkers can now make more armor. Check out the crafting menu!
rscadd: There's a new lavaland monster in town: Marrow Weaver, a venom injecting spider.
rscadd: Ashwalkers can now craft sinew tents (which protect you against ash storms) out of 3 bones, 2 sinews, 6 sandstone and 1 mithril (that you get a stack of in your base).
/:cl:

### CODE REVIEW

**Comment by Stealthkibbler** [comment]
@X-TheDark Did you remove the talk chance verb? An issue I noticed when testing with keek awhile back is the weaver chittering completely floods the ghost/chat logs since each weaver chitters every like 1-10 seconds.

As for the wrapping, this was intended so weavers would wrap up their victims and items in cocoons which would add to the mystery of "What's in the cocoon", I'm also a spider fanatic so I loved the idea of miners ripping their desiccated buddies out of encased webbing. Didnt know it would be annoying tho.

:+1:  could never figure out how to get that fucking move delay to work

you should have seen them during testing when they could lay eggs lol

as for their overrall damage, they generally wont do too much, miner armor negates a good amount of the bite damage, just the venom you gotta be careful of.

If it gets merged and actually gets some players experiencing it the health and damage could definitely need tweaking, back when I actively was working on that branch it was just me and keek testing it so we probably missed a few things. One of them I think I forgot to do was give the golems a toxin kit to be able to deal with weavers.

A thing we never got to complete was making the marrow webs actually drop bone, so to compensate the weaver just gives a lot of bones when butchered. If you or anyone else is feeling a bit ambitious on expanding the idea you could make that dream come to fruition (Which probably isnt hard I just dont know how to code).

I'm also aware that the death sprite for the weaver looks a bit bad, if you want me to do a better one I could get around to it once me and keek finish our rimworld mod or something.

Thanks for keeping this stuff alive :+1:

**Comment by X-TheDark** [comment]
No problem, @Stealthkibbler. It was probably much easier for me to remake this than it was to try to fix the merge conflicts.

Yes, I removed the stuff related to them "chittering", as it was, indeed, cluttering up the chat.

It would be cool if you could redo the icons that show in your inventory (/obj/clothing ones), as those are just a copy-paste of the mob overlay icons (they even had all 4 dir sprites defined...I fixed that).

Without the move delay (it being default), it was impossible to fight them (you couldn't run away), you'd die killing (or running away from) a single one (AshWalker spear-tested), with the basic stuff.

We'll see how this goes, it would definitely be interesting, especially with ash storms no longer being the penultimate "screw you" to ashwalkers.

**Comment by Stealthkibbler** [comment]
@X-TheDark the sprites are pretty simple, they just need to be upsized from the original front version since they use the scaling is different from actual mob sprites

File: code/game/objects/items/stacks/sheets/leather.dm [hunk 1]
```diff
@@ -145,6 +145,13 @@ var/global/list/datum/stack_recipe/sinew_recipes = list ( \
  * Plates
  		*/
 
+/obj/item/stack/sheet/animalhide/weaver_chitin
+	name = "weaver chitin"
+	desc = "a chunk of hardened and layered chitin from a marrow weavers carapace"
```
Comment (Jc1469) [inline code/game/objects/items/stacks/sheets/leather.dm]: Capital letter, fullstop. 
Sound like I'm marking school books right now.

File: code/modules/clothing/head/helmet.dm [hunk 2]
```diff
@@ -213,6 +213,16 @@
 	item_state = "skull"
 	strip_delay = 100
 
+/obj/item/clothing/head/helmet/kasa
+	name = "Pathfinders Kasa"
+	desc = "Used by AshWalkers to travel long distances across lavalands perilous landscape."
```
Comment (Jc1469) [inline code/modules/clothing/head/helmet.dm]: AshWalkers

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 3]
```diff
@@ -82,6 +82,31 @@
 	else
 		footstep++
 
+/obj/item/clothing/shoes/pathtreads
+	name = "Pathfinders Treads"
+	desc = "Massive boots crafted to protect the user from the hot ashes of lavaland"
```
Comment (Jc1469) [inline code/modules/clothing/shoes/miscellaneous.dm]: Fullstop.

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 4]
```diff
@@ -82,6 +82,31 @@
...
+	put_on_delay = 50
+	burn_state = FIRE_PROOF
+	heat_protection = FEET|LEGS
+	body_parts_covered = LEGS|FEET
+	max_heat_protection_temperature = SHOES_MAX_TEMP_PROTECT
+	can_hold_items = 1
+
+/obj/item/clothing/shoes/chitentreads
+	name = "Chitin Boots"
+	desc = "Compact boots crafted from a weavers chiten with interlacing sinew"
```
Comment (Jc1469) [inline code/modules/clothing/shoes/miscellaneous.dm]: Fullstop.

File: code/modules/clothing/suits/armor.dm [hunk 5]
```diff
@@ -105,8 +105,27 @@
 	item_state = "bonearmor"
 	blood_overlay_type = "armor"
 	armor = list(melee = 35, bullet = 25, laser = 25, energy = 10, bomb = 25, bio = 0, rad = 0)
+	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+
+/obj/item/clothing/suit/armor/chitenplate
+	name = "Chiten Plate"
+	desc = "A heavily protected and padded version of the bone armor, reinforced with chiten, sinew and bone"
```
Comment (Jc1469) [inline code/modules/clothing/suits/armor.dm]: Fullstop.

File: code/modules/clothing/suits/armor.dm [hunk 6]
```diff
@@ -105,8 +105,27 @@
...
+	icon_state = "chitenplate"
+	item_state = "chitenplate"
+	blood_overlay_type = "armor"
+	armor = list(melee = 65, bullet = 35, laser = 15, energy = 10, bomb = 35, bio = 0, rad = 0)
 	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS
 
+/obj/item/clothing/suit/armor/pathfinder
+	name = "PathFinders Cloak"
```
Comment (Jc1469) [inline code/modules/clothing/suits/armor.dm]: Either uncapitalize the F or add a space.

File: code/modules/clothing/suits/armor.dm [hunk 7]
```diff
@@ -105,8 +105,27 @@
...
+	desc = "A massive cloak that protects the wearer from concentrated burns with its layered sinew"
```
Comment (Jc1469) [inline code/modules/clothing/suits/armor.dm]: Fullstop.

File: code/modules/clothing/under/miscellaneous.dm [hunk 8]
```diff
@@ -136,6 +136,25 @@
 	can_adjust = 0
 	burn_state = FIRE_PROOF
 
+/obj/item/clothing/under/heartprot
+	name = "Heart Protector"
+	desc = "A crude piece of sinew and bone stitched together to protect ones vitals"
```
Comment (Jc1469) [inline code/modules/clothing/under/miscellaneous.dm]: one's.  Fullstop.

File: code/modules/clothing/under/miscellaneous.dm [hunk 9]
```diff
@@ -136,6 +136,25 @@
...
+	icon_state = "heartprot"
+	item_state = "heartprot"
+	can_adjust = 0
+	fitted = NO_FEMALE_UNIFORM
+	body_parts_covered = CHEST|GROIN|LEGS
+	armor = list (melee = 15, bullet = 15, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0)
+
+/obj/item/clothing/under/tatters
+	name = "Sinew Tatters"
+	desc = "The remnants of an Ash Walkers birth, still sticky and fleshy to the touch"
```
Comment (Jc1469) [inline code/modules/clothing/under/miscellaneous.dm]: Fullstop. Ash Walker's.

File: code/modules/mining/mine_items.dm [hunk 10]
```diff
@@ -272,6 +272,73 @@
 		PoolOrNew(/obj/effect/particle_effect/smoke, get_turf(src))
 		qdel(src)
 
+/*****************************Sinew Tent********************************/
+
+
+/obj/item/weapon/sinew_tent
+	name = "sinew tent"
+	desc = "a small sinew and bone box with a spring loaded tent"
```
Comment (Jc1469) [inline code/modules/mining/mine_items.dm]: :(  I have better things to do than this shit.  Plz. fix grameror

**Pull request merged**